modules/xp/graph.ts

```diff
@@ -1,86 +1,94 @@
 import { type AnySelectMenuInteraction } from "discord.js";
```

Comment:
Please revert changes in this file and put the graph conditionals in `modules/xp/index.ts`, thanks

Reply:
oo clever